compliance/tests/struct_pointer_interface_fields/struct_pointer_interface_fields.go

```diff
@@ -1,27 +1,52 @@
 package main
 
-type MyInterface interface {
-	Method()
+// Simple interface for testing
+type Reader interface {
+	Read(p []byte) (int, error)
 }
 
-type MyStruct struct {
-	PointerField   *int
-	interfaceField MyInterface
+// Struct with embedded interface
+type MyReader struct {
```

Comment:
instead of overwriting the struct_pointer_interface_fields test please make a brand new compliance test for this and checkout from origin/master compliance/tests/struct_pointer_interface_fields after